src/js/oncoprintmodel.ts

```diff
@@ -1153,7 +1154,7 @@ export default class OncoprintModel {
         }
 
         this.track_can_show_gaps[track_id] = ifndef(params.track_can_show_gaps, false);
-        this.track_show_gaps[track_id] = false;
+        this.track_show_gaps[track_id] = (params.init_show_gaps === true) ? true : false;
```

Comment:
Please simplify to:
```
this.track_show_gaps[track_id] = params.init_show_gaps;
```
:)

Reply:
Ah I did not notice that convention. Good catch.

Reply:
Done